aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1196,15 +1196,11 @@ Tensor narrow_copy_dense(const Tensor& self, int64_t dim, int64_t start, int64_t
   return self.narrow(dim, start, length).clone(at::MemoryFormat::Contiguous);
 }
 
-Tensor narrow_copy_dense_cpu(const Tensor& self, int64_t dim, int64_t start, int64_t length){
-  auto output = at::empty_like(self);
-  return narrow_copy_dense_cpu_out(self, dim, start, length, output);
-}
-
```

Comment:
Is this BC-breaking or not? Which C++ functions (if any) are public API (not documented [here](https://github.com/pytorch/pytorch/wiki/Public-API-definition-and-documentation) nor [here](https://github.com/pytorch/pytorch/wiki/PyTorch's-Python-Frontend-Backward-and-Forward-Compatibility-Policy))? Do I need to keep the old names and just dispatch to codegen'd funcs or is it fine to remove these funcs right away?

Reply:
I think it's alright. ATen functions and the C++ API are different.